packages/legend-server-sdlc/src/models/configuration/ProjectDependencyUtils.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { isString } from '@finos/legend-shared';
+
+import type { VersionId } from '../version/VersionId.js';
+
+export const buildVersionId = (version: string | VersionId): string => {
```

Comment:
why do we need this method. We can expect the result to come back as a string right?